tests/apps/gens/conftest.py

```diff
@@ -0,0 +1,23 @@
+"""Conftest for Gens."""
+
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture(name="gens_genome_build")
+def fixture_gens_genome_build() -> str:
```

Comment:
There are already defined constants here: cg/constants/gene_panel.py
```
GENOME_BUILD_37 = "37"
GENOME_BUILD_38 = "GRCh38"
```
You should use the constants here instead redefining a test fixture. However, the constant could perhaps be moved to a better module as they are not really specific to gene_panel and could be more global